micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -0,0 +1,175 @@
+    const moduleData = sortedConfigEmployeesSidebar?.[current];
+    const propsForModuleCard = {
+      // Icon: moduleData?.icon,
+      icon: "SupervisorAccount",
+      moduleName: t(moduleData?.label),
+      metrics: [],
+      links: moduleData.links,
+      centreChildren:[
+        <div>{t(Digit.Utils.locale.getTransformedLocale(`MODULE_CARD_DESC_${current}`))}</div>
+      ]    
+    };
+    return <LandingPageCard buttonSize={"medium"} {...propsForModuleCard} />;
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)